src/Core/ExecutionPathTracer/ExecutionPath.cs

```diff
@@ -123,6 +123,12 @@ public class Operation
         [JsonProperty("children")]
         public IEnumerable<IEnumerable<Operation>>? Children { get; set; }
 
+        /// <summary>
+        /// True if operation is a measurement operations.
+        /// </summary>
+        [JsonProperty("isMeasurement")]
+        public bool IsMeasurement { get; set; }
```

Comment:
Other `bool` properties in this class don't use the `Is` prefix. I'd suggest being consistent one way or the other. I personally am in favor of `Is`-like prefixes for `bool` variables, but I'll leave it up to you. (Similar comment for all the other places where these names are used.)